app/controllers/concerns/creates_bulk_actions.rb

```diff
@@ -15,7 +15,10 @@ def create
     if bulk_action.save
       bulk_action.enqueue_job(job_params)
 
-      redirect_to bulk_actions_path(Blacklight::Parameters.sanitize(search_state.to_h.except(:authenticity_token))), status: :see_other, notice: success_message
+      # strip the CSRF token, and the parameters that happened to be in the bulk job creation form
+      search_state_subset = search_state.to_h.except(:authenticity_token, :druids, :druids_only, :description)
```

Comment:
It seems to me this is a bug in blacklight. search_state.to_h should not return parameters that are not part of the search state.

Reply:
i think i agree, and i did try a couple of other methods that seemed more promising based on the name, but didn't work out (e.g. `Blacklight::SearchState#params_for_search` still included `druids`).

maybe we should leave this workaround in place with a comment till the issue is fixed upstream? 🤷

Reply:
It [looks like](https://github.com/projectblacklight/blacklight/blob/main/lib/blacklight/search_state.rb#L26-L46) `Blacklight::SearchState` does do some normalization, but I didn't see that it removed things the params provided?

Reply:
I'm inclined to merge this as is, but I defer to @jcoyne if it needs further work. One question I had for @jmartin-sul  was why preserving the parameters is needed given it's going back to the `/bulk_actions` -- are there links on that page that need the params preserved?

Reply:
Yeah, that is my question too.  Why is it preserving these parameters when it goes back to `/bulk_actions`?

Reply:
It's preserving the parameters because it needs to know what the last search was, in case you again want to click "populate with last search"

Reply:
I added a comment pointing to the Blacklight issue noting that we can remove this stripping in Argo once it is resolved there.